.circleci/config.yml

```diff
@@ -0,0 +1,47 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: ipfs/ci-websites
+        environment:
+          DEBUG: "true"
```

Comment:
All good. You can drop the DEBUG flag. That was a left over from fighting to get the build working for the docs site.

Reply:
7b8002ca462bd31942de1dc64323f36219c649d1